src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java

```diff
@@ -139,43 +139,7 @@ static void preloadJGskit(String libraryToLoad) {
                 // win64_x86
                 loadFile = new File(jgskitPath, "libjgskit_64.dll");
             }
-        } else if ((osArch.equals("aarch64")) && (osName.equals("Mac OS X"))) {
```

Comment:
I don't quite remember this part of the codes, but why we had the methods `preloadJGskit()` and `preloadOCK()` before, which have the similar functions?

Reply:
The `preloadOCK()` method loads the OCK binary (i.e, `libjgsk8iccs_64.so` or equivalent), whereas the `preloadJGskit()` method loads the OpenJCEPlus native code (i.e., `libjgskit.so` or equivalent).

Reply:
oh I see, yes, LGTM.